api/src/main/java/org/openmrs/util/DatabaseUpdater.java

```diff
@@ -199,12 +199,11 @@ public void visit(ChangeSet changeSet, DatabaseChangeLog databaseChangeLog, Data
 		log.debug("Setting up liquibase object to run changelog: " + changeLogFile);
 		Liquibase liquibase = getLiquibase(changeLogFile, cl);
 		int numChangeSetsToRun = liquibase.listUnrunChangeSets(contexts).size();
-		Database database = null;
-		LockService lockHandler = null;
+		Database database = null;. lockHandler = null;
 		
 		try {
 			database = liquibase.getDatabase();
-			lockHandler = LockService.getInstance(database);
+			lockHandler = LockServiceFactory.getInstance().getLockService;
```

Comment:
Is this compiling ? i believe **getLockService** is a method

Reply:
it isn't compiling

Reply:
replace **lockHandler = LockServiceFactory.getInstance().getLockService;** with **lockHandler = LockServiceFactory.getInstance().getLockService();** then compile the project locally ( mvn clean install )and see whether it builds without any errors before pushing your changes